pq-crypto/sike_r3/sikep434r3_ec_isogeny.c

```diff
@@ -75,9 +72,9 @@ void eval_4_isog(point_proj_t P, f2elm_t *coeff)
     mp2_add(&P->X, &P->Z, t0);                        // t0 = X+Z
     mp2_sub_p2(&P->X, &P->Z, t1);                     // t1 = X-Z
     fp2mul_mont(t0, &coeff[1], &P->X);             // X = (X+Z)*coeff[1]
-    fp2mul_mont(t1, &coeff[2], &P->Z);                // Z = (X-Z)*coeff[2]
```

Comment:
why all the changes in this file?

Reply:
I thought the comments were staggered but I think I was wrong

Reply:
from the diff it looks to me like they were aligned and now they aren't :)

Reply:
I will go back and double check but I am not sure why they look aligned on my IDE lol